dll/win32/msi/custom.c

```diff
@@ -867,6 +867,17 @@ static UINT HANDLE_CustomType1( MSIPACKAGE *package, const WCHAR *source, const
     if (!(binary = get_temp_binary(package, source)))
         return ERROR_FUNCTION_FAILED;
 
+    #if defined(__REACTOS__) && defined(_M_AMD64)
```

Comment:
please no indentation for the preprocessor things.

Reply:
Done :)